sdk/eventhub/event-hubs/test/sender.spec.ts

```diff
@@ -198,6 +198,77 @@ describe("EventHub Sender #RunnableInBrowser", function(): void {
     });
   });
 
+  describe("Create batch", function(): void {
```

Comment:
Minor: Can we make sure to close the `producers` we create at the end of each test? (I'm guilty of not doing this with producers sometimes too!)

We can do this by either calling `producer.close()` at the end of each test, or creating a variable to hold the producer inside the `describe` function's scope, and then closing it in an `afterEach` method like:

```typescript
let producer: EventHubProducer;
afterEach("close producers", async function(): Promise<void> {
  if (producer && !producer.isClosed) {
    await producer.close();
  }
  producer = undefined;
});
```